docs/docs-content/clusters/edge/cluster-management/agent-upgrade-airgap.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_label: "Upgrade Palette Agent on Airgap Clusters"
+title: "Upgrade Palette Agent on Airgap Clusters"
+description: "Learn how to upgrade the Palette agent on airgap clusters. "
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge", "architecture"]
+---
+
+In connected clusters, the Palette agent gets upgraded automatically with the Palette version, unless you explicitly
```

Comment:
Did you mean with Palette upgrades vs Palette version? Palette version seems a bit odd 😅